what4/src/What4/Solver/Yices.hs

```diff
@@ -415,13 +418,24 @@ newConnection stream ack reqFeatures bindings = do
                   .|. useStructs
                   .|. (reqFeatures .&. (useUnsatCores .|. useUnsatAssumptions))
 
-  conn <- newWriterConn stream ack nm features' bindings (Connection ())
+
+  earlyUnsatRef <- newIORef Nothing
+  conn <- newWriterConn stream (ack earlyUnsatRef) nm features' bindings (Connection earlyUnsatRef)
   return $! conn { supportFunctionDefs = True
                  , supportFunctionArguments = True
                  , supportQuantifiers = efSolver
                  }
 
-newtype YicesCommand = Cmd Builder
+data YicesCommand = YicesCommand
```

Comment:
Probably should add some code documentation here for what this safe/unsafe command stuff is all about.

Reply:
Fixed in d70caf8e.